geode-core/src/integrationTest/java/org/apache/geode/internal/cache/PartitionedRegionIntegrationTest.java

```diff
@@ -72,42 +49,10 @@ public void bucketSorterShutdownAfterRegionDestroy() {
 
   @Test
   public void bucketSorterIsNotCreatedIfNoEviction() {
-    server.startServer();
     PartitionedRegion region =
         (PartitionedRegion) server.createRegion(RegionShortcut.PARTITION, "PR1",
             rf -> rf.setOffHeap(false));
     ScheduledExecutorService bucketSorter = region.getBucketSorter();
     assertThat(bucketSorter).isNull();
   }
-
-  @Test
-  // See GEODE-7106
-  public void generatePRIdShouldNotThrowNumberFormatExceptionIfAnErrorOccursWhileReleasingTheLock() {
```

Comment:
This test doesn't make sense anymore with the new logging model?

Reply:
This test moved to PartitionedRegionTest.